dangerfile.js

```diff
@@ -1,5 +1,6 @@
 // import { danger, warn } from "danger"
 import fs from "fs"
+import includes from "lodash.includes"
```

Comment:
Do we need to worry about Node 4 support, or should I just use the built-in `Array.prototype.includes()` function here?